### PR TITLE
[Meshery] Images and descriptions for feature switcher

### PR DESCRIPTION
**Pull request by leecalcote**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Meshery/Meshery-features/index.js [hunk 1]
```diff
@@ -98,173 +108,133 @@ const MehseryFeatures = () => {
...
+                    MeshSync's working snapshot of the state of each cluster and service mesh under management is stored in-memory and continuously refreshed.
+                  </p>
+                </div>
               ),
             },
             {
               title: "Extensible Management Plane",
               description:
-                                "View projects and applications being deployed by meshery in a web interface",
+                "Plugin different service meshes, load generators and providers.",
```
Comment (leecalcote) [inline src/sections/Meshery/Meshery-features/index.js:223]:
```suggestion
                "Plug in different service meshes, load generators and providers.",
```

Comment (Nikhil-Ladha) [reply src/sections/Meshery/Meshery-features/index.js]: Reviewing your own PR....nice 😄

**Review by Nikhil-Ladha — APPROVED** [review]
lgtm

**Comment by leecalcote** [comment]
Everybody's a critic... including me. 😄

File: src/sections/Meshery/Meshery-features/index.js (hunk 1 above)
Comment (PluckyPrecious) [reply src/sections/Meshery/Meshery-features/index.js]: > Reviewing your own PR....nice 😄

😆

**Pull request merged**